anesthetic/samples.py

```diff
@@ -611,6 +611,43 @@ def dlogX(self, nsamples=None):
         else:
             return WeightedDataFrame(dlogX, self.index, weights=self.weights)
 
+    def importance_sample(self, logL_new, action='replace'):
```

Comment:
This is much better now with the `action` kwarg.

I'm willing to be argued down on this, but I think that it would be less surprising to have `action='add'` as the default, since this is what [GetDist](https://getdist.readthedocs.io/en/latest/mcsamples.html#getdist.mcsamples.MCSamples.reweightAddingLogLikes) does.

I know that your original use case was a replacement, so it might be helpful to note that replacing with `-1e30` will be the same in most cases to double precision as adding it (and will regardless have an identical effect).

Reply:
done